src/inspector/v8-debugger.cc

```diff
@@ -737,18 +737,19 @@ v8::MaybeLocal<v8::Value> V8Debugger::generatorScopes(
 }
 
 v8::MaybeLocal<v8::Array> V8Debugger::collectionsEntries(
-    v8::Local<v8::Context> context, v8::Local<v8::Value> collection) {
+    v8::Local<v8::Context> context, v8::Local<v8::Value> collection,
+    const v8::KeyIterationParams* params) {
   v8::Isolate* isolate = context->GetIsolate();
   v8::Local<v8::Array> entries;
   bool isKeyValue = false;
   if (!collection->IsObject() || !collection.As<v8::Object>()
-                                      ->PreviewEntries(&isKeyValue)
+                                      ->PreviewEntries(&isKeyValue, params)
                                       .ToLocal(&entries)) {
     return v8::MaybeLocal<v8::Array>();
   }
 
   v8::Local<v8::Array> wrappedEntries = v8::Array::New(isolate);
-  CHECK(!isKeyValue || wrappedEntries->Length() % 2 == 0);
+  CHECK(!isKeyValue || entries->Length() % 2 == 0); // replay: discovered v8 bug
```

Comment:
Good catch, this seems worth upstreaming on its own, it shouldn't be controversial.